### PR TITLE
Manual QA for MYSQL Handler is done! ✅

### PR DESCRIPTION
**Pull request by ShruAgarwal**
## Description
The manual QA for the MYSQL handler is completed successfully as stated.

**Fixes** #(issue)
It fixes issue #6219

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.

Let me know your views :)

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
@ShruAgarwal 
It seems you uploaded your data file to MindsDB. Then you connected MindsDB's files database to MindsDB itself.

Instead of doing this:
```
CREATE DATABASE mysql_test                  --- datasource connection name
WITH ENGINE = 'mysql',                      --- database engine
PARAMETERS = {
    "host": "cloud.mindsdb.com",            --- host name can be an IP address, or URL
    "port": 3306,                           --- port used to make TCP/IP connection
    "database": "files",                    --- database name
    "user": "demo-user",                    --- database user
    "password": "demo-password"             --- database password
};

SELECT * FROM mysql_test.dry_bean_data;
```

You can do this:
```
SELECT * FROM files.dry_bean_data;
```

**Please redo this manual QA by storing your data file in another MySQL database and connecting this MySQL database to MindsDB.**

File: mindsdb/integrations/handlers/mysql_handler/Manual_QA.md [hunk 1]
```diff
@@ -2,38 +2,66 @@
...
 
 **1. Testing CREATE DATABASE**
 
-```
-COMMAND THAT YOU RAN TO CREATE DATABASE.
+```sql
+CREATE DATABASE mysql_test                  --- datasource connection name
+WITH ENGINE = 'mysql',                      --- database engine
+PARAMETERS = {
+    "host": "cloud.mindsdb.com",            --- host name can be an IP address, or URL
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/mysql_handler/Manual_QA.md:13]: I see you are still using `"host": "cloud.mindsdb.com"`, and thus, connecting MindsDB to itself.

The idea is that you have your own MySQL database (here is how you can install MySQL database: https://dev.mysql.com/doc/mysql-getting-started/en/#mysql-getting-started-installing).

And then you connect this database to MindsDB:
```
CREATE DATABASE mysql_test                  --- datasource connection name
WITH ENGINE = 'mysql',                      --- database engine
PARAMETERS = {
    "host": "localhost",            --- host name can be an IP address, or URL
    "port": 3306,                           --- port used to make TCP/IP connection
    "database": "dry_bean_db",              --- database name
    "user": "demo-user",                    --- database user
    "password": "demo-password"             --- database password
};
```

If you use local database, you may need to [setup an ngrok channel](https://docs.mindsdb.com/sql/create/database#making-your-local-database-available-to-mindsdb) and use provided values for host and port.

**Comment by martyna-mindsdb** [comment]
Thanks for the updates @ShruAgarwal 

However, you still connect MindsDB to itself (as you use `"host": "cloud.mindsdb.com"` in the `CREATE DATABASE` statement).

Please see the review comment and let me know in case something is not clear.

**Comment by ShruAgarwal** [comment]
@martyna-mindsdb Should I install DBeaver and try connecting the MySQL database engine to MindsDB Cloud as per the [docs](https://docs.mindsdb.com/connect/dbeaver)?
Will this help change the "`host`" value?

**Comment by martyna-mindsdb** [comment]
@ShruAgarwal 
Here is how to install MySQL database locally: https://dev.mysql.com/doc/mysql-getting-started/en/#mysql-getting-started-installing

Once you install it, use its host/port/etc values to connect it to MindsDB.

**Comment by ShruAgarwal** [comment]
Hello, @martyna-mindsdb
You can close this PR as @gajeshbhat has been assigned this issue.
Sorry, I couldn't complete this because I was sick for the last 5 days 🙏

**Pull request closed without merging**